pyelasticsearch/tests/client_tests.py

```diff
@@ -1,7 +1,11 @@
 # coding=utf-8
 
-from mock import ANY, patch
-from nose.tools import eq_, ok_, assert_raises, assert_not_equal
+from six import PY3
+
+if PY3:
+    from unittest.mock import ANY, patch
```

Comment:
Oddly, I don't even see `patch` used anywhere, in any file. I'm not sure why we were importing it.

Reply:
Here :)
https://github.com/joehybird/pyelasticsearch/blob/72a90ab76d28721b3c134c91cb1825f6ea3d1fae/pyelasticsearch/tests/client_tests.py#L90

Reply:
Weird! Not sure how I missed it.